src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/versioncontrol/GitDecXtract.java

```diff
@@ -0,0 +1,61 @@
+package de.uhd.ifi.se.decision.management.jira.extraction.versioncontrol;
+
+import de.uhd.ifi.se.decision.management.jira.extraction.GitClient;
+import de.uhd.ifi.se.decision.management.jira.extraction.impl.GitClientImpl;
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElement;
+import org.eclipse.jgit.lib.ObjectId;
+import org.eclipse.jgit.revwalk.RevCommit;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * purpose: extract decision knowledge elements stored in git repository
+ * out-of-scope linking decision knowledge elements among each other
```

Comment:
Jochen used a simple automatic linking method that links 
- alternatives and decisions to issues 
- arguments to alternatives/decisions 
if present. The method is currently located in the JiraIssueTextPersistenceManager class.
But even without an automatic linking method, the user should be able to create new links and delete existing links between elements of any documentation location.